bootstrap-salt.sh

```diff
@@ -4474,7 +4485,7 @@ install_openbsd_deps() {
             # Let's download, since they were not provided, the default configuration files
             if [ ! -f "$_SALT_ETC_DIR/$fname" ] && [ ! -f "$_TEMP_CONFIG_DIR/$fname" ]; then
                 ftp -o "$_TEMP_CONFIG_DIR/$fname" \
-                    "https://raw.githubusercontent.com/saltstack/salt/develop/conf/$fname" || return 1
+                    "${HTTP_VAL}://raw.githubusercontent.com/saltstack/salt/develop/conf/$fname" || return 1
```

Comment:
Oh, you're right. I'll change those references back to `https` instead of using `HTTP_VAL`.